action.yml

```
@@ -17,6 +17,10 @@ inputs:
   daemon-config:
     description: 'Docker daemon JSON configuration'
     required: false
+  tcp-port:
+    description: 'TCP port to expose the Docker API locally'
+    default: 'false'
```

```suggestion
    default: 'false'
```

nit: Not sure what other options we have here, but is `false` a the best default for `number`?

---

Ok good catch we should not have any defaults here

---

Ah, it should be still fine, as that would still go into the `if (!value)` branch.

---

Yes